packages/client/src/authentication/__helpers__/signAndBroadcast.ts

```diff
@@ -0,0 +1,40 @@
+import { IEquatableError, Result } from '@lens-protocol/shared-kernel';
+import { Wallet, TypedDataDomain, TypedDataField } from 'ethers';
+
+import { getTxIdFromRelayerResult, Transaction } from '../../transaction';
+
+type TypedData = {
+  domain: TypedDataDomain;
+  types: Record<string, Array<TypedDataField>>;
+  value: {
+    nonce: number;
+    [k: string]: unknown;
+  };
+};
```

Comment:
nit: this could well go into `@lens-protocol/blockchain-bindings` and be used by both `@lens-protocol/client` and `@lens-protocol/react` packages that way.

Reply:
We have the same type defined both in `client` and in `react`, but the `client` package doesn't depend on `blockchain-bindings` nor on `api-bindings`. So, I guess, it's better to duplicate one type def than to couple a whole package.